src/config.ts

```diff
@@ -1,43 +1,55 @@
 const {
-  SERVER_PORT,
-  LCD_URI,
-  FCD_URI,
-  RPC_URI,
-  SENTRY_DSN,
-  USE_LOG_FILE,
-  DISABLE_API,
-  EXCLUDED_ROUTES,
-  MIN_GAS_PRICES,
-  PRUNING_KEEP_EVERY,
-  BANK_WALLETS,
-  TOKEN_NETWORK,
-  GENESIS_ACCOUNT_COUNT,
-  CHAIN_ID,
-  INITIAL_HEIGHT
+  INDEXER_SERVER_PORT,
+  INDEXER_CHAIN_ID,
+  INDEXER_INITIAL_HEIGHT,
+  INDEXER_LCD_URI,
+  INDEXER_FCD_URI,
+  INDEXER_RPC_URI,
+  INDEXER_SENTRY_DSN,
+  INDEXER_USE_LOG_FILE,
+  INDEXER_DISABLE_API,
+  INDEXER_EXCLUDED_ROUTES,
+  INDEXER_MIN_GAS_PRICES,
+  INDEXER_PRUNING_KEEP_EVERY,
+  INDEXER_BANK_WALLETS,
+  INDEXER_TOKEN_NETWORK,
+  INDEXER_GENESIS_ACCOUNT_COUNT
```

Comment:
@ZaradarBH can you increase the version in the package.json to indicate that breaking changes has been introduced?

Reply:
I recon I could do that. As such I believe this change is not really a breaking changes, hence a major version bump, since its the ENV variables that have changed and not the underlying configuration object that still has "sensible defaults" to ensure that it would work even when ENV variables are named wrong.  So it might be best to treat this as a minor version bump, which would put us at something like 2.1.0, sound good to you?